src/System.Management.Automation/engine/hostifaces/ConnectionBase.cs

```diff
@@ -262,6 +262,7 @@ private void CoreOpen(bool syncCall)
             OpenHelper(syncCall);
             if (etwEnabled) RunspaceEventSource.Log.OpenRunspaceStop();
 
+            // Telemetry here - runspace start
```

Comment:
no telemetry here?

Reply:
we decided to do it in PowerShell.Create instead rather than catching the runspace side.

Reply:
Can you update the comment?

Reply:
removed